StatisticsInR/2.1_simple_linear_regression.Rmd

```diff
@@ -0,0 +1,1223 @@
+
+</br>
+
+#### Cook's distance
+
+```{r, echo=T, message=F}
+plot(DiveModel, which = 4)
+```
+*Figure 9: Cook's distance plot for the dive depth model.*
+
+</br> 
+
+The assumption we are checking with this plot is: **Are there any outliers?**
+
+Figure 9 shows that the Cook's distances of this model are not very high (max = 0.2).
+So, it does not seem that any points have that large an influence on the fitted values.
+However, point 10 does seem to be quite different from the others. This might be worth 
+looking into.
+
+</br>
+
+We can find point 10 by looking at the 10th row of our data frame. It is the entry
+for the Cuvier's beaked whale. While this is a rare and unusual whale, it is not the
+only beaked whale in our dataset and we have no reason to believe this data is a typo. **Therefore, we would not consider this an outlier and would not remove from the data.**
+
+```{r, echo=T, message=F}
+DiveData[10,]
+```
+
+</br>
+
+#### Summary
+
+Overall, it seems that most of the model assumptions are met well. The only red flag is
+in the equal variance assumption. However, this seems to be caused in part by a lower amount of 
+data available at the extremes of dive depths. Therefore, we think it is ok to 
+proceed with this model.
+
+</br>
+
+In the next section we will interpret our results.
+
+</br>
+
+
+## <i class="far fa-lightbulb"></i> Draw conclusions {.tabset .tabset-fade}
+
+### Theory
+
+In the previous sections you learned how to run a simple linear regression models,
+what the parameters of the model mean, how to quantify uncertainty in the parameters, and 
+how to check the assumptions of the model. Now, we can bring everything together to draw some conclusions. 
+
+</br>
+
+There are several components required in drawing a conclusion:
+
+* statement of the maximum likelihood estimate of the parameters of interest (including strength and direction).
+* statement of the uncertainty in the estimate
+* statement of how good the model is i.e. how well the model meets assumptions and the amount of variance explained (using $R^2$ - explained below)
+* link the results to biology and the question asked
+* Discussion of next directions
+
+</br>
+
+One measure that can be useful for conclusions is to know **how much of the variation in $Y$ is explained by $X$**. To answer this we can use a measure called the $R^2$. It is calculated using the following equation:
+
+$$
+R^2 = 1 - \frac{\color{darkblue}{\text{Residual Variance}} }{\text{Total Variance}}
+$$
+and can be found in R using `summary(YOURModel)$r.squared`. The value is a proportion, so between 0 (no variance explained) and 1 (all variance explained). A good value is subjective, but > 0.5 is usually considered good, > 0.7 is very good and a value of 1 is suspicious.  
+
```

Comment:
Might be worth mentioning that for "real" biological data r squares can be much lower, because nature is just chaotic. I mean sometimes one has to be satisfied with an r squared of 0.3. Not ideal but that's real life. Not sure how much we want to explain here.